css/components/_figure.scss

```diff
@@ -13,5 +13,10 @@ figure {
     border-top: 1px solid $gray--mid;
     padding: 2px 0;
     font-size: 0.8rem;
+
+    &.thick {
```

Comment:
Ok, then I think we should just stick with the thick styles on the page. I think it improves clarity here.